### PR TITLE
improved docs

### PR DESCRIPTION
**Pull request by Sundaran13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ https://wiki.minetest.net/Installing_Mods
 
 The Minetest Game GitHub master HEAD is generally compatible with the GitHub
 master HEAD of the Minetest engine.  
-Additionally, when the Minetest engine is tagged to be a certain version (e.g.
+Additionally, when the Minetest engine is tagged to be of a certain version (e.g.
```
Comment (paramat) [inline README.md:27]: This is unnecessarily verbose, the original is fine.

**Comment by AbdullahKady** [comment]
Hey @paramat 
This seems like a poor attempt from the PR author to win a t-shirt through [hacktoberfest](https://hacktoberfest.digitalocean.com/).

If you don't mind, follow the [FAQ](https://hacktoberfest.digitalocean.com/faq) way of dealing with spam.

> **As a maintainer, how should I handle spam pull requests?**
> \+ We dislike seeing spam pull requests just as much as you, so please give them an invalid or spam label and close them

**Comment by paramat** [comment]
> This seems like a poor attempt from the PR author to win a t-shirt through hacktoberfest.

@AbdullahKady i do not doubt it but, how are you able to tell?

If so, this is related to why i have requested the 'hacktoberfest' label is removed from the MTG repo, it only attracts unwanted and unnecessary casual attention to a mostly frozen project. @rubenwardy please see #2738 

Anyway, this PR is bad English and altering one word is pointless, i will close it soon.

**Comment by AbdullahKady** [comment]
@paramat Excuse me, I was pretty frustrated with the entire thing yesterday :sweat_smile:  
Technically I can't tell, that's why I used `seems like` rather than `is`, however I feel like it's common sense given the changes proposed in the PR, and also if you check the author's account, you can see ~8 other similar PRs opened on Oct 1st in random repos (i.e. non-related topics, focusing on minimal doc changes).

Either way, stay awesome with the active maintaining :tada:

**Comment by paramat** [comment]
Hmm interesting ...

Joined GitHub in April.
No activity.
October 1st, forks 10 repos. Creates this PR and 8 others, all titled 'improve docs', all tiny grammar changes, many being bad English.
2 PRs (including this one) describe this contributor as a hacktoberfest spammer.

So i agree, thanks.

See https://hacktoberfest.digitalocean.com/details#spam , therefore labelled as 'invalid' to signal to hacktoberfest this is a spam PR so they can disqualify it.

**Comment by rubenwardy** [comment]
No need for a label for this, just mark it as invalid and close it

**Comment by paramat** [comment]
Yes, the label is unnecessary.

**Comment by paramat** [comment]
@rubenwardy please could you remove the hacktoberfest tag from the MTG repo? #2738

**Pull request closed without merging**